docs/source/NeuralNetworkTheory.rst

```diff
@@ -110,9 +110,11 @@ data.
 .. image:: ~Image/NeuralNetworkbyAndrewNg.png
 (by Andrew Ng, Machine Learning Coursera)
 
-.. image:: ~Image/RedExplicada.png
+.. image:: ~Image/RedExplicadaFinished.png
```

Comment:
Could you put the red arrows behind the black texts? They are hiding the text (for example "Layer 2").
And maybe you could add transparency to the red arrows. Thank you!

Reply:
Professor,
No problem!